R/support.R

```diff
@@ -393,3 +410,43 @@ combine_rt1 <- function(what, rt, samples) {
 
   Reduce(`+`, ret)
 }
+
+
+combine_rt1_EpiEstim <- function(what, rt, samples) {
```

Comment:
See https://github.com/mrc-ide/sircovid/pull/227 for the changes I was making to these functions before everything got backed up on the vaccination work